### PR TITLE
Spotify/demiv2.0.0👾

### PR DESCRIPTION
**Pull request by cdemiguel**
También se añade la V1 en este pull request

### CODE REVIEW

**Comment by manuelbarzi** [comment]
👍 please, ASK two reviewers more to take a look too and confirm nothing to adjust, and then we are done to merge ,)

File: spotify-app/demiv2/script/spoty-api-1.0.0.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
+/**
+ * Spotify API client.
+ *
+ * @version 1.0.0
+ */
+var spotifyApi;
```
Comment (manuelbarzi) [inline spotify-app/demiv2/script/spoty-api-1.0.0.js]: USE let instead of var here ,)

File: spotify-app/demiv2/script/main.js [hunk 2]
```diff
@@ -0,0 +1,156 @@
+(function() {
```
Comment (VGamezz19) [inline spotify-app/demiv2/script/main.js]: hey! why you need to instance all your `scopes` in a SEAF ?

File: spotify-app/demiv2/script/main.js [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+  //definimos la funcion showTracks que nos servira para enseñar las canciones
+  function showTracks(tracks) {
+
+    $(".section-album").hide();
+    $(".section-tracks").fadeIn(1000).show();
+  
+    let tracktitles = ""
+  
+    tracks.items.forEach(function(item) {
+      tracktitles+="<span class='subsection-tracks' data-url='"+item.id+"'>"+item.name+" "+"</span>";
```
Comment (VGamezz19) [inline spotify-app/demiv2/script/main.js]: yep! you can use `string literals` if you want, is more easy to read

Comment (cdemiguel) [reply spotify-app/demiv2/script/main.js]: yeah! sure, i'll do it, yesterday i didn't understand how to do, but... now with  a youtube tutorial about 20 minilessons about js6 i do!

File: spotify-app/demiv2/script/main.js (hunk 2 above)
Comment (cdemiguel) [reply spotify-app/demiv2/script/main.js]: ummmmm jajajaja sincerely i don't really know at 80% but i saw an example that was like this, if it s not necessary i will change it

File: spotify-app/demiv2/script/spoty-api-1.0.0.js [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+(function() {
+"use strict";
+
+    const token = "BQDei19I1VCMR-tBXvcUQRxoB7k0Cu4o546PJSWH766ZEvIpYUutQgs--gWmG4wOSK8bvYii4Sv2yZ896_MzA1JlYBmyC_liyEytaJcMXqFssbLsmWzi5_1q5ddw0x1r7hzI86l91OvGzAY";
+    const headers = { Authorization: 'Bearer ' + token }
+
+    function call(url, token, timeout, handleSuccess, handleError) {
+        fetch(url, { headers }).then(res => {
```
[the remark this answers is not in the document]
Comment (cdemiguel) [reply spotify-app/demiv2/script/spoty-api-1.0.0.js]: ok!! thankiu!! i will change, however i trust at 200% in spotify api !!!! muahahahaha!!

File: spotify-app/demiv2/script/spoty-api-1.0.0.js (hunk 1 above)
Comment (cdemiguel) [reply spotify-app/demiv2/script/spoty-api-1.0.0.js]: Oops! sorry, it will be changed  with the new version with all the @VGamezz19 reviews!!

**Pull request merged**